src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -510,14 +510,30 @@ private void InitializeClient(TestingClientOptions clientOptions, int largeMessa
             {
                 WriteLog("Initializing Grain Client");
                 ClientConfiguration clientConfig;
-                if (clientOptions.ClientConfigFile != null)
+
+                try
                 {
-                    clientConfig = ClientConfiguration.LoadFromFile(clientOptions.ClientConfigFile.FullName);
+                    if (clientOptions.ClientConfigFile != null)
+                    {
+                        clientConfig = ClientConfiguration.LoadFromFile(clientOptions.ClientConfigFile.FullName);
+                    }
+                    else
+                    {
+                        clientConfig = ClientConfiguration.StandardLoad();
```

Comment:
Why not do `clientConfig = ClientConfiguration.LocalhostSilo();` here instead? The file will be removed from the NuGet package and `ClientConfiguration.StandardLoad()` will start failing. Instead of catching the exception and deducing that the file wasn't there legitimately, why not move away from the implicit file approach altogether?